### ISSUE TITLE
Does the pomodoro cycle reset after a restart of chrome?

### ISSUE THREAD

**Issue by alexkolson** [schmich/marinara#87]

**Comment by schmich** [comment]
Yes, right now the Pomodoro cycle state is stored in memory and isn't persisted in any way.

The use case I had in mind included a long-running browser process, but it's easy enough to persist the state between instances.

**Comment by alexkolson** [comment]
Thank you so much for the quick response! 👍 

Personally I actually kind of like the fact that the cycle resets, but I could also see people liking it persisted.

Anyway, I love this extension! Super sleek and simple to use. Thanks for building it! It helps me be way more productive as I get distracted easily (thanks twitter!) but with this extension I can control my distractions and get stuff done.

**Comment by bbi-yggy** [comment]
A side effect is that if I leave my browser open (which I tend to do "forever") then a new day does not start a new Pomodoro cycle - but perhaps it should? It feels odd to continue yesterday's cycle. I know I can manually start a new cycle, but I would like an option to automatically start a new cycle with each new day.
